### PR TITLE
docs(java-client): update get-started

### LINKED ISSUE #2218: Update Java Get-Started guide
The "Work with Data" section in the guide needs to be updated because of the migration to a variable concept.

### PR DESCRIPTION
**Pull request by saig0**
closes #2218

### CODE REVIEW

File: docs/src/go-client/get-started.md [hunk 1]
```diff
@@ -199,14 +199,11 @@ workflowKey:1 bpmnProcessId:"order-process" version:1 workflowInstanceKey:6
 
 You did it! You want to see how the workflow instance is executed?
 
-Start the Zeebe Monitor using `java -jar zeebe-simple-monitor.jar`.
+Start the Zeebe Monitor using `java -jar zeebe-simple-monitor-app-X.Y.Z.jar`.
```
Comment (Zelldon) [inline docs/src/go-client/get-started.md]:
```suggestion
Start the Zeebe Monitor using `java -jar zeebe-simple-monitor-app-*.jar`.
```

File: docs/src/java-client/get-started.md [hunk 2]
```diff
@@ -182,14 +182,11 @@ Workflow instance created. Key: 6
 
 You did it! You want to see how the workflow instance is executed?
 
-Start the Zeebe Monitor using `java -jar zeebe-simple-monitor.jar`.
+Start the Zeebe Monitor using `java -jar zeebe-simple-monitor-app-X.Y.Z.jar`.
```
Comment (Zelldon) [inline docs/src/java-client/get-started.md]:
```suggestion
Start the Zeebe Monitor using `java -jar zeebe-simple-monitor-app-*.jar`.
```

File: docs/src/go-client/get-started.md [hunk 3]
```diff
@@ -199,14 +199,11 @@ workflowKey:1 bpmnProcessId:"order-process" version:1 workflowInstanceKey:6
...
 
 Open a web browser and go to <http://localhost:8080/>.
```
Comment (Zelldon) [inline docs/src/go-client/get-started.md]:
```suggestion

Here, you see the current state of the workflow instance which includes active jobs, completed activities, the payload and open incidents.
```

File: docs/src/java-client/get-started.md [hunk 4]
```diff
@@ -182,14 +182,11 @@ Workflow instance created. Key: 6
...
 
 Open a web browser and go to <http://localhost:8080/>.
```
Comment (Zelldon) [inline docs/src/java-client/get-started.md]:
```suggestion

Here, you see the current state of the workflow instance which includes active jobs, completed activities, the payload and open incidents.
```

File: docs/src/java-client/get-started.md [hunk 5]
```diff
@@ -356,5 +335,3 @@ Next steps:
 * Learn more about the [concepts behind Zeebe](/basics/README.html)
 * Learn more about [BPMN workflows](/bpmn-workflows/README.html)
 * Take a deeper look into the [Java client](java-client/README.html)
-
-[job worker]: ../basics/job-workers.html
```
Comment (Zelldon) [inline docs/src/java-client/get-started.md]: please readd :sweat_smile:

**Review by ChrisKujawa — CHANGES_REQUESTED** [review]
Nice gifs :+1: :smile: some smaller things

**Comment by saig0** [comment]
@Zelldon, I applied your review hints. Please have a look =)

**Review by ChrisKujawa — APPROVED** [review]

**Comment by menski** [comment]
bors retry

**Pull request merged**